apps/admin/src/app/components/user/user-status-label.tsx

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import { startCase } from 'lodash';
+import { Label, LabelColor, LabelProps } from '@libs/react-core';
+import { UserStatusEnum } from '@libs/types';
+
+export interface UserStatusLabelProps extends LabelProps {
+    label?: string;
+}
+
+const UserStatusLabel = ({ label }: UserStatusLabelProps) => {
+
+    const color: LabelColor = useMemo(() => {
+        switch (label) {
+
+            case UserStatusEnum.ACTIVE:
+                return 'primary'
+
+            case UserStatusEnum.INACTIVE:
+                return 'error'
+
+            case UserStatusEnum.PENDING:
+                return 'warning'
+
+            default:
+                return 'secondary'
```

Comment:
Default value should be `default` or null instead of `secondary`

Reply:
done